### ISSUE TITLE
Increment measures + durationNodes in IntervalInferring contexts

### ISSUE THREAD

**Issue by jsbean** [jsbean/DNM#245]
```Swift
Title: NodeEdgeTest

P: VN vn Violin sw BinarySwitch

# 3,8
+ @ VN sw
	0.125 •
	0.45 ~
	0.75 •
	1.5 ~
	4 •

+ @ VN sw
	4.25 •
```

produces:

<img width="336" alt="screen shot 2016-01-26 at 12 42 27 pm" src="https://cloud.githubusercontent.com/assets/7040073/12589140/50f23976-c42a-11e5-840d-649866553c18.png">

Issues:
- Initial measure generated is `2/8`: should be `3/8`
- Events missing?

**Comment by jsbean** [comment]
Whereas:

```Swift
Title: NodeEdgeTest

P: VN vn Violin sw BinarySwitch

# 3,8
+ @ VN sw
	0.125 •
	0.45 ~
	0.75 •
	1.5 ~
	4 •

| @ VN sw
	4.25 •
```
<img width="549" alt="screen shot 2016-01-26 at 12 44 42 pm" src="https://cloud.githubusercontent.com/assets/7040073/12589200/986cca3c-c42a-11e5-930b-eaba0d32eccb.png">

**Comment by jsbean** [comment]
```Swift
# 1,8
+ @ VN vn
	0.0 •
	1.0 •
```
produces:

<img width="231" alt="screen shot 2016-01-26 at 4 05 42 pm" src="https://cloud.githubusercontent.com/assets/7040073/12594972/af00674c-c446-11e5-952e-781188a40ef3.png">

**Comment by jsbean** [comment]
Fixed with e9a06d71813af206a88f41dc56a3b79ff24931a3
